docs/sources/features/datasources/mssql.md

```diff
@@ -269,7 +269,7 @@ GROUP BY
 ORDER BY 1
 ```
 
-When above query are used in a graph panel the result will be two series named `Metric A` and `Metric B` with an average of `valueOne` plotted over `time`.
+When above query is used in a graph panel the result will be two series named `Metric A` and `Metric B` with an average of `valueOne` plotted over `time`.
```

Comment:
```suggestion
When above query is used in a graph panel, the result is two series named `Metric A` and `Metric B` with an average of `valueOne` plotted over `time`.
```

Reply:
I have made the changes as per the @achatterjee-grafana 's suggestion